README.md

```diff
@@ -62,4 +62,5 @@ Khronos SPIR-V Registry](https://www.khronos.org/registry/spir-v/).
 1. [SPV_EXT_physical_storage_buffer         ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/EXT/SPV_EXT_physical_storage_buffer.html)
 1. [SPV_INTEL_fpga_memory_attributes        ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/INTEL/SPV_INTEL_fpga_memory_attributes.html)
 1. [SPV_NV_cooperative_matrix               ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/NV/SPV_NV_cooperative_matrix.html)
-1. [SPV_INTEL_shader_integer_functions2     ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/INTEL/SPV_INTEL_shader_integer_functions2.html)
\ No newline at end of file
+1. [SPV_INTEL_shader_integer_functions2     ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/INTEL/SPV_INTEL_shader_integer_functions2.html)
```

Comment:
Is this diff due to line-ending differences?  I think we should keep it all as linux style, but don't know the actual state right now.

Reply:
(Current repository state is linux style.)

Reply:
I don't believe this is an OS line-ending style issue.  Unix utilities and git appear to complain is there isn't a trailing \n in a file, plus show the addition of it as a modification in diff.  I believe that the previous commit must have been missing a trailing \n.  vi appends one at end of file by default, but not all editors do.

I don't think there is anything to do here.  I confirmed that my added line (new end of file) has a trailing \n in the binary dump.

Git diff says that the old commit didn't have a trailing \n:

```
git diff 4b77d36f38dee2fa95ede1a13d5b0c8c094411ba README.md
diff --git a/README.md b/README.md
index fe1f59f..a4507ce 100644
--- a/README.md
+++ b/README.md
@@ -62,4 +62,5 @@ Khronos SPIR-V Registry](https://www.khronos.org/registry/spir-v/).
 1. [SPV_EXT_physical_storage_buffer         ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/EXT/SPV_EXT_physical_storage_buffer.html)
 1. [SPV_INTEL_fpga_memory_attributes        ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/INTEL/SPV_INTEL_fpga_memory_attributes.html)
 1. [SPV_NV_cooperative_matrix               ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/NV/SPV_NV_cooperative_matrix.html)
-1. [SPV_INTEL_shader_integer_functions2     ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/INTEL/SPV_INTEL_shader_integer_functions2.html)
\ No newline at end of file
+1. [SPV_INTEL_shader_integer_functions2     ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/INTEL/SPV_INTEL_shader_integer_functions2.html)
+1. [SPV_INTEL_fpga_loop_controls            ]( http://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/INTEL/SPV_INTEL_fpga_loop_controls.html)
```

https://thoughtbot.com/blog/no-newline-at-end-of-file